portfolio/src/main/webapp/script.js

```diff
@@ -13,10 +13,10 @@
 // limitations under the License.
 
 /**
- * Adds a random quote to the page.
+ * Adds a random quote to the page as a popup.
```

Comment:
This comment (with `as a popup`) makes me think of a popup dialog, but I think this will just update the quote on the page, correct?

Reply:
yeah, i can update the description. its more of a hidden div that toggles between the randomized quotes when you click the button.